### ISSUE TITLE
Issue warning for duplicate descriptions or titles

### ISSUE THREAD

**Issue by ElliotSilver** [HL7/fhir-ig-publisher#679]
I just became aware that, likely due to a cut and paste error, an IG I'm working on has two resources with the same title on the artifacts page, and several more with the same couple of descriptions.

There may be limited reasons to have duplicate titles or descriptions (multiple versions of the same ValueSet--or an expansion of an intensional ValueSet--within one IG are an unlikely, but potentially reasonable instance).

Since duplicate titles and descriptions are not usually (but are occasionally) desired it makes sense to flag this situation as a warning.

I can't think of other elements that should also be flagged as usually be unique within an IG.

**Comment by lmckenzi** [comment]
Are you using the HL7 template?  It will belly-ache because duplicate names cause Jira issues.  The base template won't.

**Comment by ElliotSilver** [comment]
No, this isn't an HL7 template, but it's also not names that I ran into trouble with--it's titles and descriptions.

**Comment by ElliotSilver** [comment]
Consider issuing warning for descriptions, titles or names which are just copied from one of the other elements.
